.gitlab-ci.yml

```diff
@@ -37,7 +37,6 @@ stages:
   except:
     refs:
       - main
-      - tags
```

Comment:
By doing this the `check-staging-merge` and `tests-passed` tasks are also gonna run on tags. Is it your intension?